docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
-* install `nsis` and required plugin
-* install MPI
-* install MSYS2 (via `Chocolatey`) and then MinGW toolchain and required build pacakages
+
+modifying this introduction the print line `printf("\n Now It is my NEURON version!!!! \n\n");` for example: 
+
+```C
+     if (nrnmpi_myid < 1)
+        if (nrn_nobanner_ == 0) {
+            Fprintf(stderr, "%s\n", nrn_version(1));
+            printf("\nNow It is my NEURON version!!!! \n\n");
+            Fprintf(stderr, "%s\n", banner);
+            IGNORE(fflush(stderr));
+        }
+```
+and save.
+
+#### Step 14.C: Compiling the new code.
+
+Compile this new code. We can compile using Step 15. (Do it and come back here) 
+
+#### Step 14.D: Replacing the `libnrniv.dll` file
+
+It will end up creating a new `libnrniv.dll` (and `libnrnmpi_msmpi.dll` and a new `libnrnpython3.XX.dll` and new `nrniv`, but just the first one matter for THIS example) at `nrn\build\bin` (at `C:\Users\User\Neuron\nrn\build\bin` or at `C:\msys64\home\User\Neuron\nrn\build\bin` if you did the installation process here). 
+
+We can replace the old file `libnrniv.dll` at `C:\nrn\bin` for this new one and voila! We get our change implemented!! Let`s see
+
+#### Step 14.E: Checking the new `nrngui` 
+
+Run `nrngui` (inside the folder `C:\Users\User\Desktop\NEURON 9.0.0 AMD64` double click in `nrngui`)
+
+```
+NEURON -- VERSION X.X.X 2024-07-11
+
+Now it is my NEURON version!!!!
+
+Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
+See http://neuron.yale.edu/neuron/credits
+
+oc>
+```
+
+------------------------------------------------------------ 
+
+### Step 15: Compile new Neuron code
+
+
+PS: Useful commands:  
+
+```
+cd /c/Users/User/Neuron/nrn/build
+``` 
+to reach files outside of `C:\msys64\`.
+
+```
+Echo $PATH
+Which gcc 
+Which cmake 
+```
+To compile new neuron code, you can (always from a `mingw64` terminal): 
+
+#### Step 15.A: run win_build_cmake.sh 
+
+From nrn file where the source code is (example: `C:\Users\User\Neuron\nrn\ci` or `C:\msys64\home\User\Neuron\nrn\ci` if you copy a version here) run: 
+```
+.\win_build_cmake.sh
+```
+or 
+ 
+#### Step 15.B: Run "cmake .." 
+
+From nrn file where is the source code (example: C:\Users\User\Neuron\nrn or C:\msys64\home\User\Neuron\nrn if you copy a version here) run: 
+
+```
+.\cmake
+```
+
+or 
+
+#### Step 15.C: ninja
+
+Run `ninja` from build file (example: `C:\Users\User\Neuron\nrn\build` or `C:\msys64\home\User\Neuron\nrn\build` if you copy a version here) 
+```
+.\ninja
+``` 
+
+### Step 16: Dependences
+
+Python will not have all dependences, so you can install them doing:  
+
+```
+python -m pip install plotly 
+python -m pip install matplotlib 
+python -m pip install pandas 
+``` 
+
+
+## Creating a new environment for neuron development 
+It my be useful maintain the original version of neuron installed (at  for example) and have a new version that you can do your changes in code. We will set up this new environment and version.
```

Comment:
looks like you're missing something in `(at  for example)`

Reply:
done